core/build.gradle

```diff
@@ -108,6 +108,7 @@ dependencies {
     optionalApi libs.micrometer.prometheus
     optionalApi libs.dropwizard.metrics.core
     optionalApi libs.prometheus
+    implementation 'io.micrometer:context-propagation:1.1.1'
```

Comment:
- Could you specify the version numbers in /dependeicies.toml, not here?

Reply:
Thanks for your comment 🙇‍♂️ 
i make new commit to follow your guideline!